content/departments/engineering/dev/tools/grpc_cs_enablement.md

```diff
@@ -0,0 +1,110 @@
+  - [zoekt-indexserver](https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/release/base/indexed-search/indexed-search.StatefulSet.yaml)
+  - [symbols](https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/release/base/symbols/symbols.Deployment.yaml)
+  - [repo-updater](https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/release/base/repo-updater/repo-updater.Deployment.yaml)
+- The following aspects about Sourcegraph’s networking configuration **aren’t changing**:
+  - \***\*\*\*\*\***Ports:\***\*\*\*\*\*** all Sourcegraph services will use the same ports as they were in the **5.1.X** release.
+  - **External traffic**: gRPC only affects how Sourcegraph’s microservices communicate amongst themselves - **no new external traffic is sent via gRPC**.
+  - **Service dependencies:** each Sourcegraph service will communicate with the same set of services regardless of whether gRPC is enabled.
+    - Example: `searcher` will still need to communicate with `gitserver` to fetch repository data. Whether or not gRPC is enabled doesn’t matter.
+
+## enabling / disabling GRPC
+
+gRPC will be the only means of communication used in Sourcegraph `5.3.X`. In the `5.2.x` release, you can disable the use of gRPC by:
+
+### all services besides zoekt-indexserver
+
+You can either:
+
+- set the `enableGRPC` experimental feature to `false` in the site configuration file:
+  ```json
+  {
+    "experimentalFeatures": {
+      "enableGRPC": false // disabled
+    }
+  }
+  ```
+- set the environment variable `SG_FEATURE_FLAG_GRPC="false"` for every service.
+
+### zoekt-indexserver
+
+_(zoekt-indexserver can’t read from Sourcegraph’s site configuration, so we can only use environment variables to communicate this setting)._
+
+Set the environment variable `GRPC_ENABLED="false"` on the zoekt-indexserver container. (See [https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/18e5f9e450878705b7a99ee7c3bcf74c3fb68514/base/indexed-search/indexed-search.StatefulSet.yaml#L105-L106](https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/18e5f9e450878705b7a99ee7c3bcf74c3fb68514/base/indexed-search/indexed-search.StatefulSet.yaml#L105-L106) for an example:
+
+```yaml
+- name: zoekt-indexserver
+          env:
+            - name: GRPC_ENABLED
+              value: "false"
+          image: docker.io/sourcegraph/search-indexer:5.2.0
+```
+
+## How do you know if there might be a problem with gRPC?
+
+_See the gRPC [monitoring guide](https://handbook.sourcegraph.com/departments/engineering/dev/tools/grpc/#grpc-monitoring-guide) for more background._
+
+It’s important to understand that gRPC is merely the transport mechanism that our entire application logic is built on top of. Our existing monitoring, alerting, and reliance on user-submitted bug reports are our best tools for identifying issues with the underlying application.
+
+However, we do have two tools that can help identify issues with gRPC.
```

Comment:
Nitpick `identify issues` is a different task than finding out `if there might be a problem` (which is the header).